_posts/2018-04-03-CocoaPods-1.5.0.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "CocoaPods 1.5.0 &#8212; Swift Static Libraries"
+author: segiddins
+categories: cocoapods releases
+---
+
+_CocoaPods 1.5.0_ comes with native support for building Swift pods as static libraries.
+
+<!-- more -->
+
+Just a few months after the release of CocoaPods 1.4.0, we're releasing a new version that focuses on enabling _everyone_ to adopt Swift.
+
+### Swift Static Libraries
+
+Up until Xcode 9, support for building Swift into static libraries was not-existent and use of dynamic frameworks was required. This was a deal-breaker for some developers, particularly those worried about the launch performance implications of linked many dynamic binaries.
+
+With CocoaPods 1.5.0, developers are no longer restricted into specifying `use_frameworks!` in their `Podfile` in order to install pods that use Swift. Interop Objective-C should also just work. If your Swift pod depends on an Objective-C pod you will need to enable modular headers.
```

Comment:
@segiddins I think this sentence needs to be better/more accurate "Interop Objective-C should also just work. If your Swift pod depends on an Objective-C pod you will need to enable modular headers."

Feel free to edit.